core/imageroot/var/lib/nethserver/node/actions/add-public-service/50add

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
+
+#
+# Copyright (C) 2022 Nethesis S.r.l.
+# http://www.nethesis.it - nethserver@nethesis.it
+#
+# This script is part of NethServer.
+#
+# NethServer is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License,
+# or any later version.
+#
+# NethServer is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NethServer.  If not, see COPYING.
+#
+
+import sys
+import json
+import agent
+
+request = json.load(sys.stdin)
+name = request['service']
+ports = request['ports']
+
+fw_cmd = ['firewall-offline-cmd']
+
+# Create the service
+agent.run_helper(*fw_cmd, f'--new-service={name}').check_returncode()
+
+# Add the service to default zone
+agent.run_helper(*fw_cmd, f'--add-service={name}').check_returncode()
```

Comment:
The public zone is the default one. I'd like to assume it always work on that zone.

Reply:
I know that `public` is the default zone, but I'd still like to have it explicit: we can't assume all machines from cloud provider are configured in the same way.
This shouldn't be a big change, we should add the zone only here and inside the 60wgboot file.

Reply:
I think it does not matter what is the default zone. Cloud providers can select what they want (why though?): I see no reason to change it.

NS8 has no Network settings page, whilst other softwares like NetworkManager, Kickstart, Cockpit, Cloud-init might want to control network settings: I want to avoid conflicts on that field, so I think it's better to leave the default system settings untouched and assume nobody wants to touch it during the system lifetime.

Reply:
Still not convinced, but ok :)